rockethost.go

```diff
@@ -185,7 +177,7 @@ func (a *App) StartRHostMatch(settings RHostMatchSettings) (string, error) {
 	}
 
 	err = conn.SendPacket(&flat.ConnectionSettingsT{
-		AgentId:              "rlbot/gui",
+		AgentId:              "",
```

Comment:
It used to be this way before but iirc, that resulted in the gui not receiving a fieldinfo packet once a match starts.

Reply:
I tested. This works with RLBotServer v0.4.8, which is publishing now

Reply:
Sounds good, I trust you